### DISCUSSION TITLE
Welcome to the Yarra Support Forum 👋

### DISCUSSION THREAD

**Discussion by tblock79** [yarra-framework/yarra-framework.github.io#1] [Announcements]
### Welcome to the new Yarra Support Forum 

Please use this place to ask questions and make suggestions for the Yarra Framework. The former Gitter-based Yarra Support Forum will not be used anymore.

**Comment by skwarfield** [comment]
I would like to configure the Yarra RDS client on a scanner running XA30.
The client provides the option to select one of the mounted network drives to set the transfer location.
It also provides a space for the 'net use' command.

We can mount the drive we want for the network target when we use the 'net use' command in a terminal run in user or administrator modes.
We can also mount the network drive using Windows explorer.  However, the drive is still not then available for selection by Yarra.

Is there an additional step we need to do to make the drive accessible by Yarra ?

**Reply by tblock79** [reply]
Hmm, that's strange. I am not aware that we had any such issues here. We've been using it on a range of software versions, including XA30. What's your installation path? We install it typically in c:\yarra

We run the "net use" command as part of the "Startup Commands". Thus, our network share is always connected (to drive v:\ in our case). 

If you can't select the path in the UI, can you try what happens if you edit it directly in the file rds.ini? Below are the required settings (replace anything PUT_... with the values that apply to your setup)

[Network]
Mode=0
DriveBasepath=PUT_PATH_HERE (e.g. V:/Data)
DriveReconnectCmd=net use v: \\\\PUT_SERVER_IP\\PUT_PATH PUT_PASSWORD /USER:PUT_USERNAME /PERSISTENT:YES /yes
DriveCreateBasepath=true
DriveStartupCmdsAfterFail=true

**Comment by skwarfield** [comment]
Thank you.  We were able to connect and transfer data after we entered the right information directly in the rds.ini file.

**Comment by hyunsoolee0503** [comment]
Dear Tobias, I have another question regarding the similar issue posted on Q&A. My colleague tried to install yarra RDS on XA50 system and he failed. I told him how to change the smartscreen settings and he changed properly and could push 'run' button after medadmin login. However, after medadmin login, another error message was popped up saying " Your Organization used Windows Defender Application Control to block this app". Do you happen to have any suggestions to solve this issue? Best regards, Hyun-Soo
